src/backend/distributed/commands/multi_copy.c

```diff
@@ -1234,6 +1234,11 @@ ReportCopyError(MultiConnection *connection, PGresult *result)
 		/* probably a constraint violation, show remote message and detail */
 		char *remoteDetail = PQresultErrorField(result, PG_DIAG_MESSAGE_DETAIL);
 
+		if (remoteDetail == NULL)
+		{
+			remoteDetail = "(null)";
+		}
+
 		ereport(ERROR, (errmsg("%s", remoteMessage),
 						errdetail("%s", remoteDetail)));
```

Comment:
Postgres sometimes uses: 

``` 
(remoteDetail != NULL) ? errdetail("%s", remoteDetail) : 0,
```

that way the detail is hidden instead of showing up as "DETAIL: (null)"

Reply:
Nice, that's much better.